lib/pages/chat_list/chat_list.dart

```diff
@@ -71,6 +73,21 @@ class ChatListController extends State<ChatList>
 
   String? activeSpaceId;
 
+  late final profileMemoizers = <Client?, AsyncMemoizer<Profile>>{};
+
+  Future<Profile?> fetchOwnProfile({required Client client}) {
+    if (!profileMemoizers.containsKey(client)) {
+      profileMemoizers[client] = AsyncMemoizer();
+    }
+    return profileMemoizers[client]!.runOnce(() async {
+      return await client.fetchOwnProfile();
+    });
+  }
```

Comment:
IMO, before your change, `client.fetchOwnProfile()` only call one. Why we need `AsyncMemoizer`?